### ISSUE TITLE
Facing some issue in deploying latest docker image

### ISSUE THREAD

**Issue by ram2104** [SolidStateGroup/bullet-train-api#1289]
While deploying the latest docker image `Digest: sha256:a62a2ae8d2427be8504b017a6cabf66d6bd79e7f6e32ac2f0baef96c950ae698`, I'm facing this issue.

```
   with self.db.wrap_database_errors:
  File "/usr/local/lib/python3.10/site-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.OperationalError: cannot ALTER TABLE "features_feature" because it has pending trigger events
```

<img width="995" alt="Screenshot 2022-07-08 at 5 44 35 PM" src="https://user-images.githubusercontent.com/19301579/177990495-b1d5938d-6491-48f3-987a-c416e7f48b03.png">

**Comment by dabeeeenster** [comment]
Running:

```bash
docker-compose pull
docker-compose up
```

Works ok for me with a clean, new DB. This is against `flagsmith/flagsmith:latest` which currently is hash `8e0a5ed94090`

**Comment by dabeeeenster** [comment]
Can you confirm the Postgres version you guys are running?

How feasible would it be for you to get us a copy of your database?

**Comment by ram2104** [comment]
Hey @dabeeeenster, I was kind of doing the exploring the flagsmith therefore I just off the data in the Postgres DB to unblock myself.

But I was kind of reading online. Maybe this may help us.
https://stackoverflow.com/questions/12838111/django-db-migrations-cannot-alter-table-because-it-has-pending-trigger-events

**Comment by matthewelwell** [comment]
Yes, that is why we'd like to know the version of postgres you are using. I believe that the issue you experienced shouldn't happen after postgres 9.4.

**Comment by ram2104** [comment]
And also on the slight note for another issue, I was also facing this as the issue.

**app.views    WARNING  Invalid request made to /flagsmith/health with method HEAD**

**Comment by ram2104** [comment]
We were facing this issue because of the Postgres version. Thanks for sharing it. Happy to close it.
